### ISSUE TITLE
ESM error in Node.js 16

### ISSUE THREAD

**Issue by thelinuxlich** [a-synchronous/rubico#195]
```
import { map, tryCatch, pipe, reduce, curry, filter } from "rubico"
                                      ^^^^^
SyntaxError: Named export 'curry' not found. The requested module 'rubico' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

import pkg from 'rubico';
const { map, tryCatch, pipe, reduce, curry, filter } = pkg;
```

**Comment by richytong** [comment]
could you try import * as rubico from "rubico"?

**Comment by thelinuxlich** [comment]
This works:
```
import rubico from 'rubico'

const { map, tryCatch, pipe, reduce, curry, filter } = rubico;
```
Also, the es.js file when imported gives the same error

**Comment by richytong** [comment]
all right, i'll look into it

**Comment by richytong** [comment]
Okay, should be fixed in 1.8.2 (035c279). You should be able to import rubico in ESM Node 16 as

```
import { pipe, tap, map } from "rubico/dist/rubico.mjs"
```

**Comment by richytong** [comment]
rest of the functions .mjs in 1.8.3
